src/mapping/rentalManager.ts

```diff
@@ -121,7 +124,7 @@ export function parseFunction(
         error,
         blockNumber: transaction.block.height,
         blockHash: transaction.block.hash,
-        address: consts.CONTRACT_ADDRESS[chain][consts.CONTRACT.RENTAL_MANAGER],
+        address: contractAddress,
```

Comment:
Was wondering whether we could use `transaction.to` here. Would shave off a parameter.

Reply:
will leave this unresolved. haven't looked at this, we can incorporate in the future PRs